exercises/07-automate-deployment.md

```diff
@@ -1,49 +1,50 @@
 # Automate deployment
-In a real world scenario you would only deploy your changes, once they have been tested for quality. You would execute the tests and checks as well as the deployment automatically. Therefore, we provide the [SAP Cloud SDK Continuous Delivery Toolkit](https://github.com/SAP/cloud-s4-sdk-pipeline) in addition to the libraries that you have seen before.
-
-This toolkit allows you to easily setup and configure a continuuous delivery pipeline for Jenkins. It also supports Travis CI (less feature rich) and GitHub Actions (experimental).
-
-We will use Github Actions as a lightweight solution today.
+In a real-world scenario, you would only deploy your changes, once they have been tested for quality. You would execute the tests and checks as well as the deployment automatically. We will use GitHub Actions as a lightweight solution in this tutorial.
 
 ## Review the CI / CD configuration
+
 Let's take a look at some configuration files:
-* [pipeline_config.yml](../pipeline_config.yml): Configure the steps of your pipeline.
-* [build.yml](../.github/workflows/build.yml): Arrange steps for your pipeline.
 
-If you use the SAP Cloud SDK Continuous Delivery Toolkit in a productive environment you will probably rather use a Jenkins server as of today. For this you would load the pipeline through a Jenkins file and would not need to arrange your steps in the `build.yml`.
+- [.pipeline/config.yml](../.pipeline/config.yml): Configure the steps of your pipeline.
+- [.github/workflows/build.yml](../.github/workflows/build.yml): Arrange steps for your pipeline.
+
+In a production environment, you will probably rather use a Jenkins server as of today. For this, you would load the pipeline through a Jenkins file and would not need to arrange your steps in the `build.yml`.
 
 ## Adjust pipeline configuration
-Go to the [pipeline_config.yml](../pipeline_config.yml) file and adjust it to your project. Set the org and space for the `cloudFoundryDeploy` step as you have done before when logging in to Cloud Foundry (if you don't remember run `cf target` on the command line to get a summary).
-Also, replace your participantId for the `appName`.
+
+Go to the [.pipeline/config.yml](../.pipeline/config.yml) file and adjust it to your project. Set the org and space for the `cloudFoundryDeploy` step as you have done before when logging in to Cloud Foundry (if you don't remember, run `cf target` on the command line to get a summary).
 
 ## Set credentials for deployment
+
 Take a look at the `build.yml` file. The `Deploy` step will deploy your application to the SAP BTP. This step needs credentials.
-To set your credentials in Github go to  "Settings" and then "Secrets":
+To set your credentials in Github go to "Settings" and then "Secrets":
```

Comment:
[req] consider the style guide for the go to steps.

Reply:
follow [this style guide](https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#placeholders) and changed the go to step